index.php

```diff
@@ -23,7 +23,12 @@
 		<?php
 		error_reporting(0);
 		$file="/tmp/babeldump";
-		shell_exec('echo "dump" | nc ::1 33123 -q 1 > '.$file.'');
+
+		if (filemtime($file) < time()-60) {
+			shell_exec("rm $file && touch $file");
```

Comment:
Die Zeile können wir weg lassen. Das erledigt die redirection in der nächsten Zeile.